Framework/Algorithms/src/ExtractMask.cpp

```diff
@@ -80,10 +80,6 @@ void ExtractMask::exec() {
   }
   PARALLEL_CHECK_INTERUPT_REGION
 
-  // Clear all the "masked" bits on the output masked workspace
-  Geometry::ParameterMap &pmap = maskWS->instrumentParameters();
-  pmap.clearParametersByName("masked");
-
```

Comment:
Looks like the documentation was lying about what this algorithm was doing. It would be nice to fix the commented out unit test, but it can probably be left to get picked up by the `SpectrumInfo` rollout.